includes/nf/ziti.h

```diff
@@ -125,7 +125,7 @@ typedef struct nf_options_s {
     nf_init_cb init_cb;
     nf_service_cb service_cb;
 
-    int refresh_interval;
+    long refresh_interval;
```

Comment:
what made you change this into a long? is this in millis?

Reply:
no, it just cleared a warning